exercise-integration-testing-5-53/sensors-api.js

```diff
@@ -13,7 +13,7 @@ const initializeAPI = () => {
     })
   );
   expressApp.use(bodyParser.json());
-  expressApp.listen();
+  // expressApp.listen();
```

Comment:
I prefer it when you let the production code open the connection.

Why? Because then there must be some code somewhere that does this and you didn't include it in your tests. What you test is not 100% the production code (99% or so). The app currently won't start in production... 

Yes, I didn't emphasize this enough during the lesson